website/docs/cluster-management/generic-clusters.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Generic Clusters
+hide_title: true
+sidebar_position: 2
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Generic Cluster <TierLabel tiers="enterprise" />
+
+### Managing non-capi clusters
+
+Any kubernetes cluster whether capi or not can be added to Weave Gitops Enterprise.
+
```

Comment:
Describe a scenario where you have just created a terraform k8s cluster.
- If you have a kubeconfig as a result of this process describe how to add it to a secret and then create a `GitopsCluster`.
- If you don't have a kubeconfig or a short lived one then get ppl to follow the below instructions. Then add a `GitopsCluster` object.